src/config/configuration.js

```diff
@@ -1,4 +1,5 @@
 const assert = require('assert');
+const CorbadoApi = require("../services/CorbadoApi");
```

Comment:
datei nicht commited. grundsätzlich sollte in der config nichts initialisiert werden, die config sollte irgendwo hin übergebne werden.